### DISCUSSION TITLE
Analysing Quarkus Native startup RSS consumption

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#35052] [Quarkus Blog/Website]
# Analysing Quarkus Native startup RSS consumption

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/native-startup-rss-troubleshooting/

**Comment by bulldog98** [comment]
Interesting to see how the memory regression in native mode was discovered. Also a fan of showing how to use jfr to do that in newer quarkus versions.
